<launch>
    <node name="map_segmentation" pkg="task_3" type="map_segmentation.py" output="screen">
        <param name="segmentation_size" type="int" value="1" />
        <param name="min_quad_size" type="int" value="15" />
    </node>

    <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="log" />
    <node name="tts" pkg="task_3" type="tts.py" output="log" args="$(find task_3)/data/ $(find task_3)/data/ris-2021-e782e57981e3.json"></node>
    <node name="mover" pkg="task_3" type="mover.py" output="screen">
        <param name="replay_waypoints" value="true" />
        <param name="rotate_on_replay" value="true" />
    </node>

    <node name="cylinder_segmentation" pkg="task_3" type="cylinder_segmentation" output="log">
        <remap from="input" to="/camera/depth/points" />
    </node>
    <node name="cylinder_handler" pkg="task_3" type="cylinder_handler.py" output="log" />

    <node name="color_classifier" pkg="task_3" type="color_classifier.py" output="log" args="$(find task_3)/data/" />

    <node name="ring_segmentation" pkg="task_3" type="ring_segmentation.py" output="screen" />


    <node name="move_arm" pkg="task_3" type="move_arm.py" output="log" />
</launch>
